server/src/main/java/org/opensearch/index/store/lockmanager/RemoteStoreMetadataLockManager.java

```diff
@@ -88,6 +88,11 @@ public String fetchLock(String filenamePrefix, String acquirerId) throws IOExcep
         return lockFilesForAcquirer.get(0);
     }
 
+    public Collection<String> fetchLocks(String filenamePrefix) throws IOException {
```

Comment:
nit : Should we rename to `fetchLockedFiles` as we are not returning `LockInfo`, but names of files which are locked .

Reply:
Kept it consistent with `fetchLock` method in the same class.